tutorials/wordle.md

```diff
@@ -550,27 +545,18 @@ compile the blockchain and take it out for a test drive.
 ## ⛓️ Run the wordle chain
 <!-- markdownlint-disable MD013 -->
 
-### 🪶 Run a Celestia light node {#run-celestia-light-node}
-
-Follow instructions to install and start your Celestia Data Availability
-layer Light Node selecting the Arabica Devnet. You can
-find instructions to install and run the node [here](https://docs.celestia.org/nodes/light-node).
+### 🪶 Run a mock DA node {#run-mock-da-node}
```

Comment:
btw these `{#headings}` are for custom anchors, where for example, the emoji would make the linkable heading ugly. https://vitepress.dev/guide/markdown#custom-anchors